governance/dashboard-update.py

```
@@ -0,0 +1,292 @@
+- 在制卡：**{len(cards)}** 张（active 仓 open+state:*）
+{state_lines}
+- factory-floor 板：{payload["project"]["url"] or "（board-sync 首轮后回填链接）"}
+- SLI（#98 口径，v1 子集）：自动合并率 {rate_txt} · 卡死 PR（>24h）{sli['stuck_prs']}
+- 待补（W5-C3）：人类触碰/PR · 门禁逃逸率 · 假红率 · 熵增——见 sli_pending
+- 刷新节奏：butler-ledger 每 15min（唤醒矩阵行 2）；手动：workflow_dispatch board-sync
+"""
+    return human
+
+
+def ensure_issue(body):
+    """幂等找到/创建账本 issue；返回 (number, created)。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)